call_apply_bind_exercise/callApplyBind.js

```diff
@@ -0,0 +1,136 @@
+// Fix the following code:
+
+var obj = {
+    fullName: "Harry Potter",
+    person: {
+        sayHi: function(){
+            return "This person's name is " + this.fullName
+        }
+    }
+}
+obj.person.sayHi.call(obj)
+
+
+
+var obj = {
+    fullName: "Harry Potter",
+    sayHi: function(){
+        return "This person's name is " + this.fullName
+    }
+}
+
+
+
+
+
+
+// Write a function called sumEvenArguments which takes all of the arguments passed to a function and returns the sum of the even ones.
+// sumEvenArguments(1,2,3,4) // 6
+// sumEvenArguments(1,2,6) // 8
+// sumEvenArguments(1,2) // 2
+
+//I feel like there should be an easier way to do this with less code ....
+function sumEvenArguments() {
+	var arr = [].slice.call(arguments)
+	var evens = []
+	arr.forEach(function(element) {
+		if (element % 2===0) {
+			evens.push(element)
```

Comment:
instead of pushing all of the evens into a new array, you could just do the sum here.